examples/nnp_training.py

```diff
@@ -80,37 +80,17 @@
 except NameError:
     path = os.getcwd()
 dspath = os.path.join(path, '../dataset/ani1-up_to_gdb4/ani_gdb_s01.h5')
-
 batch_size = 2560
 
-training, validation = torchani.data.load_ani_dataset(
-    dspath, species_to_tensor, batch_size, rm_outlier=True, device=device,
-    transform=[energy_shifter.subtract_from_dataset], split=[0.8, None])
+dataset = torchani.data.load(dspath).subtract_self_energies(energy_shifter).remove_outliers().species_to_indices().shuffle()
```

Comment:
I think we can take out the default `remove_outliers()` from example files. The ANI1x and ANI2x datasets now have the outlier energy conformers removed. And in the case of custom datasets, the user may come up with different thresholds than defaults.

Reply:
fixed